documentation/configure.markdown

```diff
@@ -620,6 +620,18 @@ tuned based on what you see in the performance dashboard.][dashboard]
 
 This setting defaults to half the number of cores in your system.
 
+### `concurrent-writes`
+
+This sets a limit on the number of threads that can write to the disk at
+any one time. The default value is the smaller number of half the number
+of CPU cores and 4.
+
+If your load is low, your disk is fast (i.e. an SSD), and commands
+aren't being processed quickly enough, then you could increasing this
+value in order to alleviate that, but this is unlikely to be the
+bottleneck for command processing. Note that any value higher than that
+for the command processing `threads` parameter has no effect.
```

Comment:
My misunderstanding. Updated to remove this line and tweak the config.ini comment to not imply this.